R/types.R

```diff
@@ -685,7 +685,7 @@ is_copyable <- function(x) {
 is_equal <- function(x, y) {
   identical(x, y)
 }
-#' @useDynLib rlang rlang_is_identical
-is_identical <- function(x, y) {
-  .Call(rlang_is_identical, x, y)
+#' @useDynLib rlang rlang_is_reference
+is_reference <- function(x, y) {
```

Comment:
@hadley Just renamed `is_identical()` to `is_reference()`. I think that function can be generally useful and that name feels right. Would it be ok to export it?